### PR TITLE
Add keptrial test

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

**Comment by barentsen** [comment]
Ohhh Travis seems to dislike the plotting.

**Comment by mirca** [comment]
@barentsen yeah =/

**Comment by mirca** [comment]
cool, all green, merging

**Comment by barentsen** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ before_install:
   - pip install pytest-cov
 install:
   - pip install -r requirements.txt
+before_script: # configure a headless display to test plot generation
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```
Comment (barentsen) [inline .travis.yml]: I think we'll want to get rid of this eventually.  Pyke tools should be able to run on machines without displays!

**Pull request merged**